includes/wikia/api/ArticlesApiController.class.php

```diff
@@ -1351,6 +1351,21 @@ public function getExcludeNamespacesFromCategoryMembersDBQuery() {
 	}
 
 	/**
+	 * Get Main Page Article ID for passed Wiki by its ID
+	 */
+	public function getMainPageForWiki() {
+		$this->getResponse()->setFormat( WikiaResponse::FORMAT_JSON );
+		$this->response->setCacheValidity( self::SIMPLE_JSON_VARNISH_CACHE_EXPIRATION * 30 );
+
+		$wikiId = $this->request->getVal( 'wikiId', null );
+		if ( !empty( $wikiId ) ) {
+			$mainPageTitle = WikiFactory::getVarValueByName( 'wgSitename', $wikiId );
+			$mainPageId = GlobalTitle::newFromText($mainPageTitle, NS_MAIN, $wikiId)->getArticleID();
+			$this->response->setVal('mainPageArticleId', $mainPageId);
```

Comment:
it should be `article_id` or `main_page_article_id` (probably the former, as method is called `getMainPage...` instead of `mainPageArticleId`